modules/modify-policy-categories-using-tab.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-security-policies.adoc
+:_content-type: PROCEDURE
+[id="modify-policy-categories-using-tab_{context}"]
+= Modifying policy categories using the Policy categories tab
+
+{product-title-short} version 3.74 provides a new method to create and manage policy categories in {product-title-managed} or in {product-title-short} if you have the PostgreSQL database (Technology Preview) enabled. All policy workflows other than policy creation remain unchanged when using this feature. For instructions on using this feature, see the second procedure in the following section.
+
+You can also configure policy categories by using the `PolicyCategoryService` API object. For more information, navigate to *Help* -> *API reference* in the {product-title-short} portal. 
+
+:FeatureName: PostgreSQL support
+include::snippets/technology-preview.adoc[]
+
+.Procedure
+
+To edit or delete policy categories in {product-title-managed} or if you have the PostgreSQL database feature (Technology Preview) enabled:
```

Comment:
Per the SSG, we shouldn't be adding lead-in sentences to procedure steps. Any info like this should be moved up to the intro section. (Same for the other new module)